rtf_vis_tool/src/stylesheets/FrontendSummary.css

```diff
@@ -0,0 +1,48 @@
+/* Stylings for FrontendSummary.js Component.
+
+Author: Adi Singh
+*/
+
+/* Styles the div containing all the frontend summary metrics text. Gives a tan background with surrounding black
+   border. Located in the first quadrant of the screen. 
+*/
+.fs_container {
+    position: absolute;
+    top: 20%;
+    right: 10%;
+    z-index: 3;
+    border: 3px solid black;
+    background-color: #fcaf3e;
+    width: 20%;
+    height: 40%;
+    border-radius: 5px;
+    padding: 1%;
+}
+
+/* Styles subcontainer div to take all the space within .fs_container. Has relative positioning. */
```

Comment:
do we need all of these different css classes? or can they be shared?

Reply:
For the most part, I think we'll need different css classes since each css class specifies a group of stylings for a specific JS component. For example, if we look at `FrontendSummary.css` and `MVOSummary.css`, despite the two pop-ups looking the same, each container has different positioning offsets from the top and left side of the screen as well as different heights and widths. So even in this case, we'd need separate stylings for those two.